cli/internal/client/client.go

```diff
@@ -211,3 +213,97 @@ func (c *APIClient) addTeamParam(params *url.Values) {
 		params.Add("slug", c.teamSlug)
 	}
 }
+
+// JSONPatch sends a byte array (json.marshalled payload) to a given endpoint with PATCH
```

Comment:
Maybe worth renaming this, as I was expecting https://jsonpatch.com/

Reply:
do you have a better name? I considered just `Post` and `Patch`, but since there are other methods here that use, `application/octect-stream` for example, I wanted to be clear that I'm not attempting to co-opt.